frontend/src/__tests__/cypress/cypress/utils/projects.ts

```diff
@@ -0,0 +1,11 @@
+import { projectListPage } from '~/__tests__/cypress/cypress/pages/projects';
+
+/**
+ * Filter Project by name using the Project filter from the Data Science Projects view
+ * @param projectName Project Name
+ */
+export const filterProjectByName = (projectName: string) => {
+    projectListPage.findProjectsDropdownFilter().click();
+    projectListPage.findProjectsDropdownFilterButtonByText("Name").click();
+    projectListPage.findProjectsFilterInput().type(projectName);
+};
```

Comment:
This should be part of the project list page object as a helper function.

Reply:
@christianvogt I've moved it to the same file as the project lists. Is that OK or should I do something different?

3f4d1ed